apps/mobile/src/features/ConfirmTx/components/confirmation-views/RemoveSigner/utils.tsx

```diff
@@ -0,0 +1,36 @@
+import { SafeFontIcon } from '@/src/components/SafeFontIcon'
+import { Logo } from '@/src/components/Logo'
+import { Text, View } from 'tamagui'
+
+import { Chain } from '@safe-global/store/gateway/AUTO_GENERATED/chains'
+import { Identicon } from '@/src/components/Identicon'
+import { getSignerName } from '../AddSigner/utils'
+
+import { NormalizedSettingsChangeTransaction } from '../../ConfirmationView/types'
+
+export const formatRemoveSignerItems = (txInfo: NormalizedSettingsChangeTransaction, chain: Chain) => {
+  const newSignerRemoveress = getSignerName(txInfo)
```

Comment:
What's a "removeress"? New removed signers?

Reply:
removeress is a typo 😆 it should be `newRemovedSigners`